atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/transaction/impl/TransactionTestSetup.java

```diff
@@ -125,73 +127,78 @@ protected TransactionManager getManager() {
                 transactionService, conflictDetectionManager, sweepStrategyManager);
     }
 
-    protected void put(Transaction t,
+    protected void put(Transaction txn,
             String rowName,
             String columnName,
             String value) {
-        put(t, TEST_TABLE, rowName, columnName, value);
+        put(txn, TEST_TABLE, rowName, columnName, value);
     }
 
-    protected void put(Transaction t,
+    protected void put(Transaction txn,
             TableReference tableRef,
             String rowName,
             String columnName,
             String value) {
-        Cell k = Cell.create(PtBytes.toBytes(rowName), PtBytes.toBytes(columnName));
-        byte[] v = value == null ? null : PtBytes.toBytes(value);
-        HashMap<Cell, byte[]> map = Maps.newHashMap();
-        map.put(k, v);
-        t.put(tableRef, map);
+        Cell cell = Cell.create(PtBytes.toBytes(rowName), PtBytes.toBytes(columnName));
+        byte[] valueBytes = value == null ? null : PtBytes.toBytes(value);
+        Map<Cell, byte[]> map = Maps.newHashMap();
+        map.put(cell, valueBytes);
+        txn.put(tableRef, map);
     }
 
-    protected void delete(Transaction t, String rowName, String columnName) {
-        t.delete(TEST_TABLE, ImmutableSet.of(Cell.create(PtBytes.toBytes(rowName), PtBytes.toBytes(columnName))));
+    protected void delete(Transaction txn, String rowName, String columnName) {
+        txn.delete(TEST_TABLE, ImmutableSet.of(Cell.create(PtBytes.toBytes(rowName), PtBytes.toBytes(columnName))));
     }
 
-    protected String get(Transaction t,
+    protected String get(Transaction txn,
```

Comment:
I was required to switch it around so that overloaded methods were together. Agreed that it could have been done in separate commits.